notebooks/feature_serving.py

```
@@ -0,0 +1,244 @@
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Call The Endpoint
+
+# COMMAND ----------
+
+
+# COMMAND ----------
+
+token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff

Undefined name `dbutils`

(F821)